src/dissect/dissect.c

```diff
@@ -127,6 +132,14 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
                 }
 
+                case ARG_HASH_DEVICE: {
```

Comment:
The {} is unnecessary here as this block doesn't need any locally scoped variables